### PR TITLE
Map page skeleton

### LINKED ISSUE #3940: Create MAP Skeleton Page
**Is your feature request related to a problem? Please describe.**
We need a skeleton page to place our MAP content

**Describe the solution you'd like**
We are currently re-using the "MediaPage" for our MAP routes.  We need this to be a different page such that, when we create new components for it, we can add those components to this page, and build it out over time.

The skeleton page can just have some placeholder content for now

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Automated tests were already created as part of the initial routing work - update them if required.

- [ ] This feature is expected to need manual testing.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by ibMadbouly**
Resolves #3940 
this PR is building the skeleton for the `MAP` pages

**Code changes:**
- at `src/app/containers` added the skeleton for `CpsAsset` pages 
   - `src/app/containers/CpsAssetPage` to be the main Container
   - `src/app/containers/CpsAssetPageMain` to carry the layout of the `CpsAssetPage` and renders a placeholder content

** Testing Notes **
  just make sure the `CPS` routing is working fine for `canonical` ,`amp` also the `.json`

- `http://localhost:7080/punjabi/international-49567825`
- `http://localhost:7080/punjabi/international-49567825.amp`
- `http://localhost:7080/punjabi/international-49567825.json`
Screen Shot 2019-10-08 at 2 15 57 PM

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

**Comment by pjlee11** [comment]
This has a lot of issues linked and most of them aren’t being resolved. Different from the current radioPage the CpsAssetPage will not need it’s own blocks. As the issues describe they should render using the Blocks container and therefore the existing articles containers. Therefore could you remove the issues that aren’t about creating a MAP page. Then can you always remove the “CpsAssetPageBlocks”. 

Any questions let me know

File: src/app/containers/CpsAssetPageMain/index.test.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import CpsAssetPageMain from '.';
+import pidginPageData from '#data/pidgin/cpsAssets/tori-49450859';
+import addIdsToBlocks from '../../routes/getInitialData/radioPage/addIdsToBlocks';
+
+const pageData = addIdsToBlocks(pidginPageData);
```
Comment (pjlee11) [inline src/app/containers/CpsAssetPageMain/index.test.jsx]:
```suggestion
```

not needed

File: src/app/containers/CpsAssetPageMain/index.test.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import CpsAssetPageMain from '.';
+import pidginPageData from '#data/pidgin/cpsAssets/tori-49450859';
+import addIdsToBlocks from '../../routes/getInitialData/radioPage/addIdsToBlocks';
```
Comment (pjlee11) [inline src/app/containers/CpsAssetPageMain/index.test.jsx]:
```suggestion
```

File: src/app/containers/CpsAssetPage/index.test.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+const cpsAssetScaffoldProps = {
+  isAmp: false,
+  pageType: 'media',
+  service: 'pidgin',
+  pathname: '/pathname',
+  match: {
+    params: {
+      serviceId: 'cpsAssets',
+      mediaId: 'tori-49450859',
```
Comment (pjlee11) [inline src/app/containers/CpsAssetPage/index.test.jsx]:
```suggestion
      assetUri: 'tori-49450859',
```

File: src/app/containers/CpsAssetPageMain/index.jsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    <>
+      <Grid as="main" role="main">
+        <GridItemConstrainedMedium>
+          <h1> Placeholder content for MAP page skeleton</h1>
+        </GridItemConstrainedMedium>
+      </Grid>
+    </>
+  );
+};
+/* eslint-disable */
```
Comment (pjlee11) [inline src/app/containers/CpsAssetPageMain/index.jsx]: Why is this disable needed?

File: src/app/containers/CpsAssetPageMain/index.jsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    promo: shape({
+      id: string,
+      type: string,
+    }),
+    content: shape({
+      blocks: arrayOf(
+        shape({
+          uuid: string,
+          id: string,
+          externalId: string,
```
Comment (pjlee11) [inline src/app/containers/CpsAssetPageMain/index.jsx]:
```suggestion
```

`externalId` is only needed on the radio page

File: src/app/containers/CpsAssetPageMain/index.test.jsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+const pageData = addIdsToBlocks(pidginPageData);
+
+describe('CpsAssetPageMain', () => {
+  shouldMatchSnapshot(
+    'should match snapshot',
+    <ServiceContextProvider service="news">
```
Comment (pjlee11) [inline src/app/containers/CpsAssetPageMain/index.test.jsx]:
```suggestion
    <ServiceContextProvider service="pidgin">
```

You've used pidgin data so the service should be consistent

File: src/app/containers/CpsAssetPageMain/index.test.jsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+      <RequestContextProvider
+        bbcOrigin="https://www.test.bbc.co.uk"
+        isAmp={false}
+        pageType="media"
+        pathname="/pathname"
+        service="news"
```
Comment (pjlee11) [inline src/app/containers/CpsAssetPageMain/index.test.jsx]:
```suggestion
        service="pidgin"
```

File: src/app/containers/CpsAssetPageMain/index.test.jsx [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+const pageData = addIdsToBlocks(pidginPageData);
+
+describe('CpsAssetPageMain', () => {
+  shouldMatchSnapshot(
+    'should match snapshot',
+    <ServiceContextProvider service="news">
+      <RequestContextProvider
+        bbcOrigin="https://www.test.bbc.co.uk"
+        isAmp={false}
+        pageType="media"
```
Comment (pjlee11) [inline src/app/containers/CpsAssetPageMain/index.test.jsx]:
```suggestion
        pageType="MAP"
```

File: src/app/containers/CpsAssetPage/index.jsx [hunk 9]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import compose from 'ramda/src/compose';
+
+import CpsAssetPageMain from '../CpsAssetPageMain';
+
```
Comment (pjlee11) [inline src/app/containers/CpsAssetPage/index.jsx]:
```suggestion
```

File: src/app/containers/CpsAssetPage/index.jsx [hunk 10]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import compose from 'ramda/src/compose';
+
```
Comment (pjlee11) [inline src/app/containers/CpsAssetPage/index.jsx]:
```suggestion
```

File: src/app/containers/CpsAssetPage/index.test.jsx [hunk 11]
```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import CpsAssetPage from '.';
+import pidginPageData from '#data/pidgin/cpsAssets/tori-49450859';
+
+const cpsAssetScaffoldProps = {
+  isAmp: false,
+  pageType: 'media',
```
Comment (pjlee11) [inline src/app/containers/CpsAssetPage/index.test.jsx]:
```suggestion
  pageType: 'MAP',
```

**Review by pjlee11 — CHANGES_REQUESTED** [review]
Changes please 🙂

File: src/app/containers/CpsAssetPageMain/index.jsx [hunk 12]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { string, shape, object, arrayOf } from 'prop-types';
+import { Grid, GridItemConstrainedMedium } from '#lib/styledGrid';
+
+const CpsAssetPageMain = () => {
+  return (
+    <>
```
Comment (ryanmccombe) [inline src/app/containers/CpsAssetPageMain/index.jsx]: Don't need this wrapping <> </>

Comment (ibMadbouly) [reply src/app/containers/CpsAssetPageMain/index.jsx]: Done 👍

**Review by ryanmccombe — APPROVED** [review]

File: src/app/containers/CpsAssetPage/index.test.jsx [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+import pidginPageData from '#data/pidgin/cpsAssets/tori-49450859';
+
+const cpsAssetScaffoldProps = {
+  isAmp: false,
+  pageType: 'MAP',
+  service: 'pidgin',
+  pathname: '/pathname',
+  match: {
+    params: {
+      serviceId: 'cpsAssets',
```
Comment (pjlee11) [inline src/app/containers/CpsAssetPage/index.test.jsx]:
```suggestion
```

`serviceId` was part of radioPage

**Review by pjlee11 — CHANGES_REQUESTED** [review]
Looking good, one minor change

File: src/app/containers/CpsAssetPage/index.test.jsx (hunk 13 above)
Comment (ibMadbouly) [reply src/app/containers/CpsAssetPage/index.test.jsx]: Done ☕️

**Review by pjlee11 — APPROVED** [review]
LGTM 👍

File: src/app/containers/CpsAssetPageMain/index.jsx [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+    <Grid as="main" role="main">
+      <GridItemConstrainedMedium>
+        <h1> Placeholder content for MAP page skeleton</h1>
+      </GridItemConstrainedMedium>
+    </Grid>
+  );
+};
+
+CpsAssetPageMain.propTypes = {
+  /* eslint-disable react/no-unused-prop-types */
```
Comment (rhenshaw56) [inline src/app/containers/CpsAssetPageMain/index.jsx:16]: question, why are we disabling prop-type warnings here ?

File: src/app/containers/CpsAssetPageMain/index.jsx [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+  pageData: shape({
```
Comment (rhenshaw56) [inline src/app/containers/CpsAssetPageMain/index.jsx:17]: question, is there no way we could reuse most of the proptypes in `#models/`? like [here](https://github.com/bbc/simorgh/blob/latest/src/app/models/propTypes/radioPage/index.js#L6)

File: src/app/containers/CpsAssetPageMain/index.jsx (hunk 14 above)
Comment (ibMadbouly) [reply src/app/containers/CpsAssetPageMain/index.jsx:16]: got linting error locally `18:13  error  'pageData' PropType is defined but prop is never used  react/no-unused-prop-types`, better to disable this rule for now `  /* eslint-disable react/no-unused-prop-types */ `

**Review by rhenshaw56 — APPROVED** [review]
Looks good!

**Review by jroebu14 — APPROVED** [review]

**Comment by LilyL0u** [comment]
Looks good using the links to punjabi MAPs.

Canonical:

Image 09-10-2019 at 12 42

AMP:

Image 09-10-2019 at 12 42

JSON:

Image 09-10-2019 at 12 43

**Pull request merged**